package/libs/openssl/Config.in

```diff
@@ -297,11 +297,14 @@ config OPENSSL_ENGINE
 		This enables alternative cryptography implementations,
 		most commonly for interfacing with external crypto devices,
 		or supporting new/alternative ciphers and digests.
+		Note that you need to enable KERNEL_AIO to be able to build the
+		afalg engine package.
```

Comment:
Please move this hunk to commit _openssl: add configuration options, disable ssl3_